tsconfig.json

```diff
@@ -24,5 +24,8 @@
     "strictBindCallApply": true,
     "target": "es5",
     "types": ["jest", "node"],
+    "paths": {
+      "@amplitude/analytics-node": ["./packages/analytics-node/"]
```

Comment:
This is to allow the example to use the `@amplitude/analytics-node` without extra installation, also make life easier to debug local changes without extra `yarn link`.

Reply:
was hoping the examples would be using published versions so they'd be more reliable. idk what do you think?

Reply:
Good call! I slightly prefer this way than the published versions, but I am good with both. The reasons I slightly prefer this way:
- easier for community developer to test their change locally, before opening PRs here
- react-native has a well-known issue when linking local dependency, this will make symlink not work. Seems the only easy way to test local change is to use this way, see the [legacy react-native](https://github.com/amplitude/Amplitude-ReactNative/blob/main/tsconfig.json#L5). I haven't test this on react-native example though, but I know `yarn/npm link` is not working for me (@yuhao900914 shared this tsconfig solution to me). Note, the published versions are working smoothly, but we will likely to have local uncommitted changes all the time for development purpose.
  - https://stackoverflow.com/questions/44061155/react-native-npm-link-local-dependency-unable-to-resolve-module
  - https://github.com/facebook/metro/issues/1

As you mention, the published versions are more stable, which will the examples more stable too. @kevinpagtakhan , let me know if you prefer the published versions, I can update to use it.

Reply:
Update to use the published versions to be aligned with other examples in this repo, we can probably revisit this for all the SDKs, for example Kotlin Android is using the local version: https://github.com/amplitude/Amplitude-Kotlin/blob/main/samples/kotlin-android-app/build.gradle#L45.